### PR TITLE
Make doc structure consistent and up-to-date

### PR DESCRIPTION
**Pull request by Julian-O**
This is part of an effort to make the Kivy sibling projects' documentation structure consistent and up-to-date.

[Bringing the lessons learnt back to the top.]

CHECKLIST

* CONTRIBUTING.md [X] If repo takes user contributions, is present [X] In root dir (not .github dir) [X] Explains relationship to Kivy, if unclear. [NA] Refers to kivy/kivy Contribution Guidelines.

* LICENSE [X] If repo takes user contributions, is present. [X] Acknowledges the range of years to 2023. [X] Acknowledges Kivy Team and other contributors [X] Mentions it is an MIT License.

* CODE_OF_CONDUCT.md [X] If repo takes user contributions or hosts discussions, is present. [NA] Refers to kivy/kivy CODE_OF_CONDUCT.md

* CONTACT.md [NA] Refers to kivy/kivy CONTACT.md

* FAQ.md [X] If repo is big enough for RST documentation, is present.

* README: [X] Is a Markdown file. [X] Describes the project. [X] Describes its place in the Kivy sibling projects. [X] If Documentation exists, mention it. [X] If CONTRIBUTING exists, mentions it. [X] If LICENSE exists, mentions it. [X] If CODE_OF_CONDUCT exists, mentions it. [X] Mentions kivy/kivy CONTACT.md [NA] Uses Python syntax colouring for embedded Python code. [] Uses badges to display current status, including: [X] Backers [X] Sponsors [X] GitHub contributors [X] Contributor Covenant [X] PyPI Version [X] PyPI Python Version [X] Build/Test status

   [X] Displays all contributors to the repo.
   [X] Displays backers
   [X] Displays top sponsors.

* RST documentation, if present [X] Describes the project. [???] Describes its place in the Kivy sibling projects. [X] Mentions (Kivy/Kivy) Contact Us link. [X] Mentions LICENSE. [X] Mentions CONTRIBUTING [X] Mentions FAQ [X] conf.py mentioned Kivy Team and other contributors
		- copyright, latex_documents, man_pages, texinfo documents

* WORKFLOWS [X] NO_RESPONSE.yml is present if the repo has awaiting_reply tag. [X] NO_RESPONSE uses latest script versions. [X] NO_RESPONSE runs every day, and skips if forked. [X] SUPPORT.yml is present if the repo has a `support` tag. [X] SUPPORT.yml refers to repo's CONTACT.md

* setup.py/cfg, if present and on PyPI [X] Supplies description to PyPI [X] Supplies Python versions to PyPI [X] Supplies Documentation, if any, to PyPI

Maintainer merge checklist
* [ ] Title is descriptive/clear for inclusion in release notes.
* [ ] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [ ] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

File: doc/sources/conf.py [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
 # General substitutions.
 project = 'Kivy'
-copyright = '2010, The Kivy Authors'
+copyright = '2010-2024, 2024, Kivy Team and other contributors'
```
Comment (misl6) [inline doc/sources/conf.py:69]: The next  release will be performed in 2023, so better to keep 2023 here.

**Review by misl6 — CHANGES_REQUESTED** [review]
Hi @Julian-O ,
With the copyright date set to 2023, everything looks fine.

File: doc/sources/conf.py (hunk 1 above)
Comment (Julian-O) [reply doc/sources/conf.py]: @misl6 Done! Thanks.

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**